datalad/support/gitrepo.py

```diff
@@ -419,13 +419,13 @@ def process_exited(self):
             )
         super().process_exited()
 
-    def pipe_data_received(self, fd, byts):
+    def pipe_data_received(self, fd, bytes):
```

Comment:
The reason for the previous name was to avoid shadowing the built-in `bytes`. I still consider it better style to not reuse built-in names, even though this particular could bit might be short enough to realize that this is not a type.

Reply:
I agree